lib/rubocop/config.rb

```diff
@@ -710,7 +710,10 @@ def enable_cop?(qualified_cop_name, cop_options)
         end
       end
 
-      cop_options.fetch('Enabled', true)
+      cop_options.fetch('Enabled') do
+        disabled_by_default = for_all_cops['DisabledByDefault']
```

Comment:
I guess you can simplify this to simply `!for_all_cops['DisabledByDefault']`.

Reply:
👍 I opted for the verbose version because it looked nicer.